.github/workflows/muelu_clang_format.yml

```diff
@@ -0,0 +1,18 @@
+name: Check MueLu clang-format
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: DoozyX/clang-format-lint-action@v0.16.2
+      with:
+        source: './packages/muelu/src'
```

Comment:
Run it directly from `muelu`?

Reply:
How?

Reply:
Oh you mean `./packages/muelu` instead of `./packages/muelu/src`?